### PR TITLE
Update to not use deprecated index and constraint syntax

### PR DESCRIPTION
**Pull request by Hunterness**
They are currently in progress of being removed and are therefore in need of being updated

### CODE REVIEW

**Review by HannesSandberg — CHANGES_REQUESTED** [review]
I have problem with building this branch, maybe you could help me out.

Found some more places to update:

`CREATE CONSTRAINT ON ... ASSERT` in `SchemasTest.java`

`db.index.fulltext` in `SignatureTest.java` and `signatures.csv`

File: core/src/main/java/apoc/export/cypher/MultiStatementCypherSubGraphExporter.java [hunk 1]
```diff
@@ -231,7 +231,8 @@ private void exportSchema(PrintWriter out) {
     private List<String> exportIndexes() {
         return db.executeTransactionally("CALL db.indexes()", Collections.emptyMap(), result -> result.stream()
```
[the remark this answers is not in the document]
Comment (Hunterness) [reply core/src/main/java/apoc/export/cypher/MultiStatementCypherSubGraphExporter.java:232]: no this method is the one causing https://trello.com/c/KJUMznPU/2751-apoc-failures-due-to-removal-of-dbindexes and can not yet be removed without breaking things

**Comment by Hunterness** [comment]
> `db.index.fulltext` in `SignatureTest.java` and `signatures.csv`

is it the create or query methods? cause it is only create/drop that are being removed, the query ones are still valid

File: core/src/main/java/apoc/export/cypher/MultiStatementCypherSubGraphExporter.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (HannesSandberg) [reply core/src/main/java/apoc/export/cypher/MultiStatementCypherSubGraphExporter.java:232]: Ok. I see

**Comment by HannesSandberg** [comment]
> > db.index.fulltext in SignatureTest.java and signatures.csv

> is it the create or query methods? cause it is only create/drop that are being removed, the query ones are still valid

Ok, these are not create so should be fine then

**Comment by conker84** [comment]
Hi @Hunterness what is the status of this one?

**Comment by Hunterness** [comment]
@conker84 Not really sure, I don't know why it doesn't build but I would like to see a green build before merging to know I have fixed things properly (I had issues with building things locally in general so I can't even check it myself)

**Review by HannesSandberg — APPROVED** [review]
Lgtm. I get a few test failures when building this locally. But as far as I can tell it has nothing to do with these changes.

**Comment by Hunterness** [comment]
@conker84 I managed got get it to build locally, and all tests I have changed seems to work now so I'll merge and hope nothing else breaks

**Pull request merged**